test/src/test/java/jenkins/security/ClassFilterImplTest.java

```diff
@@ -149,6 +151,16 @@ public void xstreamRequiresWhitelist() throws Exception {
         assertEquals(Collections.singleton(config), data.keySet());
         assertThat(data.values().iterator().next().extra, allOf(containsString("LinkedListMultimap"), containsString("https://jenkins.io/redirect/class-filter/")));
     }
+
+    @Test
```

Comment:
`@WithoutJenkins`?

Reply:
It would not be reliable just in case ClassFilterImpl tests require extension points & Co later